docs/content/test/InProcessClick.md

```diff
@@ -0,0 +1,35 @@
+---
+title: In process click
+type: docs
+---
+
+# In process click
+
+It is click without inter-process communication.
+
+## Motivation to write custom click
+
+Espresso click use inter-process communication. It leads to problems:
+- Flaky clicks on moving views. Because click takes some time to touch after coordinates calculating.
+- "misinterpret clicks as long clicks"
+
+## Implementation details
+
+Click happens via dispatch touch event on root view
+
+### Known issues
+
+- Can click through any system elements on the screen. It applies clicks directly on root
```

Comment:
Can we show this in a test?
I'm worried about duplicating implementation details in the documentation.

Reply:
I don't know. I tired of this task. Everything i do is move text from source to documentation.